docs/validators/overview.md

```diff
@@ -7,11 +7,11 @@ title: Validators - Overview
 
 ### Install and maintain a node
 
-| Name                                                                            | Description                                                              |
```

Comment:
update also the link on L13 or remove completely

Reply:
removed